.buildkite/pipeline.yml

```diff
@@ -19,10 +27,31 @@ steps:
   - wait: ~
     continue_on_failure: true
 
-  - label: "Junit annotate"
+  - label: ":pipeline: Trigger Integration tests"
```

Comment:
Should we remove integration tests from the Jenkinsfile?

Reply:
At first, in this PR I wanted to check that both Jenkins and Buildkite, had the same results.
I've created this follow-up PR to remove all those stages related to tests once this is merged
https://github.com/elastic/elastic-package/pull/1147